jmh-core/src/main/java/org/openjdk/jmh/profile/AsyncProfiler.java

```diff
@@ -92,7 +97,7 @@ public AsyncProfiler(String initLine) throws ProfilerException {
                 .withRequiredArg().ofType(String.class).describedAs("path");
 
         OptionSpec<String> optEvent = parser.accepts("event",
-                "Event to sample: cpu, alloc, wall, lock, cache-misses, etc.")
+                "Event(s) to sample: cpu, alloc, lock, wall, itimer; com.foo.Bar.methodName; any event from `perf list` e.g. cache-misses")
```

Comment:
So, instead of making `event` a multi-option, my suggestion is to add `alloc` and `lock` as separate options that will be directly translated to async-profiler's peer options.

Reply:
This sounds good to me. I'll implement it this way.